Answers/jijonj - HW03_turtle.py

```diff
@@ -0,0 +1,79 @@
+###################################################
+##  Author: Julio Jijon
+##  Username: jijonj
+##  Google Doc: https://docs.google.com/document/d/1lS28AuVN5m-JRi-ODe0WctUhKYdrz50QzcrktSkBx60/edit?usp=sharing
+##  Purpose: create a house with a person using turtles
+###################################################
+
+import turtle
+
+def drawSquare(t, size):
+    ''' put in a for loop for a square'''
+    for i in range(6):
```

Comment:
This loops 6 times, but a square has four side...

If you need to reposition the turtle after the loop ends, that should happen outside of the function (i.e., the function should draw squares, because that's what the function's job is to do; in `main()`, you are building a house, so position should happen there)